src/components/__tests__/EmptyThought.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import App from '../../App'
+
+// eslint-disable-next-line fp/no-let
+let wrapper = null
+
+beforeAll(async () => {
+  wrapper = await window.createTestApp(<App />)
```

Comment:
Can `createTestApp` and `cleanupTestapp` be imported directly, or do they have to be attached to `window`?

Reply:
Done